srcpkgs/ly/files/ly/run

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+tty=${PWD##*-}
+
+[ -r conf ] && . ./conf
+
+if [ -x /sbin/getty -o -x /bin/getty ]; then
+	# busybox
+	GETTY=getty
+elif [ -x /sbin/agetty -o -x /bin/agetty ]; then
+	# util-linux
+	GETTY=agetty
+fi
+
+exec setsid ${GETTY} ${GETTY_ARGS} -nl /usr/bin/ly tty7 "${BAUD_RATE}" "${TERM_NAME}"
```

Comment:
tty7 shouldn't be hardcoded.

Reply:
What would be the correct way to set it? Subsitute `tty7` with `${tty}`?

Reply:
no, because the service is being installed as `ly`. It would need to be installed as `ly-tty7` for the `tty=${PWD##*-}` trick to work.

Reply:
It's hardcoded to tty2 in the original run file. I'm starting to think that I'm going to need to scrap a lot of the work done in the void fork on the service files.

Reply:
Actually, now that I think about it, is it possible to read in data from another file in the conf file?

Reply:
Ideally no, the conf file should be reasonably simple

Reply:
What are you thinking of doing?

Reply:
I'm considering `grep`ing the `/etc/ly/config.ini` file for the tty number and using that in the arguments passed to `/usr/bin/ly`.

Reply:
As a side note, can I move the getty vs agetty thing in the run file into the conf file?

Reply:
Honestly, I don't have any issue with making this package depend on `util-linux` directly, so imo you can just use `agetty`.

Reply:
> I'm considering greping the /etc/ly/config.ini file for the tty number and using that in the arguments passed to /usr/bin/ly.

I prefer the user configuring this manually, with you leaving sensible defaults. Otherwise it becomes a little too magical.

Reply:
Just changed it. Could you take a look at the conf, run, and config.ini.patch files to see if that's sufficient to allow users to understand what they need to do?